### PR TITLE
Add P1 Components.

### PR DESCRIPTION
**Pull request by dhruvin25799**
Added P1 Components.
Check components.css after Input for P1 Components

### CODE REVIEW

File: components.css [hunk 1]
```diff
@@ -76,14 +76,14 @@
...
 
-.alert .alert-dismiss{
-    color: white;
-    margin-left: auto;
-    cursor: pointer;
+.alert .alert-dismiss {
+  color: white;
+  margin-left: auto;
+  cursor: pointer;
 }
```
Comment (manojsarna) [inline components.css]: use variables for colors , eg --COLOR-BLACK

File: components.css [hunk 2]
```diff
@@ -76,14 +76,14 @@
...
 
-.alert .alert-dismiss:hover{
-    color: black;
+.alert .alert-dismiss:hover {
+  color: black;
 }
```
Comment (manojsarna) [inline components.css]: same as above

File: components.css [hunk 3]
```diff
@@ -420,3 +420,204 @@ input:valid ~ label {
...
+/* Chip */
+.chip {
+  display: inline-block;
+  padding: 0.5rem 1rem;
+  border-radius: 1rem;
+  background-color: white;
+  border: 1px solid var(--btn-primary-red);
+  display: flex;
+  flex-flow: row nowrap;
+  align-items: center;
```
Comment (manojsarna) [inline components.css]: if values are atmost 10 px , can be used in px

File: components.css [hunk 4]
```diff
@@ -420,3 +420,204 @@ input:valid ~ label {
...
+
+/* Navigation Bar */
+
+.nav {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 30px 10%;
+  background-color: white;
+}
```
Comment (manojsarna) [inline components.css]: rems can be used

File: components.css [hunk 5]
```diff
@@ -420,3 +420,204 @@ input:valid ~ label {
...
+
+.nav a {
+  color: black;
+  transition: all 0.3s ease 0s;
+}
```
Comment (manojsarna) [inline components.css]: check color

File: components.css [hunk 6]
```diff
@@ -420,3 +420,204 @@ input:valid ~ label {
...
+
+.nav a:hover {
+  color: red;
+}
+
```
Comment (manojsarna) [inline components.css]: color check, use variables

File: components.css [hunk 7]
```diff
@@ -420,3 +420,204 @@ input:valid ~ label {
...
+  font-size: larger;
+}
+
+.nav .nav-links li a {
+  transition: all 0.3s ease 0s;
+  text-decoration: none;
+  color: black;
+}
+
+.nav .nav-links li a:hover {
```
Comment (manojsarna) [inline components.css]: color check , use variables

File: components.css [hunk 8]
```diff
@@ -420,3 +420,204 @@ input:valid ~ label {
...
+  position: fixed;
+  z-index: 99999;
+  padding-top: 100px;
+  left: 0;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  overflow: auto;
+  background-color: rgba(0, 0, 0, 0.4);
+}
```
Comment (manojsarna) [inline components.css:529]: check color , use variables if possible

File: components.css [hunk 9]
```diff
@@ -420,3 +420,204 @@ input:valid ~ label {
...
+  font-size: larger;
+  color: #aaaaaa;
+}
+
+.modal .modal-body .modal-dismiss:hover {
+  cursor: pointer;
+  text-decoration: none;
+  color: black;
+}
+
```
Comment (manojsarna) [inline components.css]: check colors

File: components.css [hunk 10]
```diff
@@ -420,3 +420,204 @@ input:valid ~ label {
...
+.s2:hover ~ a {
+  color: red;
+}
+.s3:hover ~ a {
+  color: red;
+}
+.s4:hover ~ a {
+  color: red;
+}
+.s5:hover ~ a {
```
Comment (manojsarna) [inline components.css]: use variables in colors

**Review by manojsarna — APPROVED** [review]
overall code is good. 
using varibales will help you tackling future changes!!

Looks good to merge!!

File: components.css [hunk 11]
```diff
@@ -4,6 +4,9 @@
   --warning: #fde68a;
   --info: #17a2b8;
   --white: white;
+  --black: black;
```
Comment (dev-ashishsethi) [inline components.css:7]: you could have used "color" keyword before color name to make it easy while searching for colors

File: components.css [hunk 12]
```diff
@@ -292,7 +295,7 @@
 
 .card .card-badge-container .card-badge {
   position: absolute;
-  background-color: green;
+  background-color: var(--success);
   padding: 0 1%;
```
Comment (dev-ashishsethi) [inline components.css:299]: percent can be a issue instead use rem

File: components.css [hunk 13]
```diff
@@ -420,3 +423,204 @@ input:valid ~ label {
...
+}
+
+.nav .nav-links li {
+  display: inline-block;
+  padding: 0 20px;
+  font-size: larger;
+}
+
+.nav .nav-links li a {
+  transition: all 0.3s ease 0s;
```
Comment (dev-ashishsethi) [inline components.css]: I suggest you to use 0.2s instad of 0.3s

**Review by dev-ashishsethi — APPROVED** [review]
added some suggestions otherwise looks good

**Pull request merged**